protocols/identify/src/behaviour.rs

```diff
@@ -431,6 +431,8 @@ pub enum Event {
     Pushed {
         /// The peer that the information has been sent to.
         peer_id: PeerId,
+        /// The information pushed to the peer.
```

Comment:
Should we maybe add some docs here to say that this is the _full_ `Info` we currently know about ourselves and that users need to do diffing themselves if that are interested in _what_ changed?